src/cachers/base.js

```diff
@@ -30,6 +30,9 @@ class Cacher {
 			keygen: null,
 			maxParamsLength: null
 		});
+
+		/** @type {boolean} Flag indicating the connection status */
+		this.connected = false;
```

Comment:
Maybe `this.connected` should be `undefined` by default on the base cacher.

This would avoid breaking any Custom-Cachers currently in use, if the check below check if `this.connected` was `undefined && false`

Reply:
@AndreMaz please initialize this to `null` and set `this.connected = false` in Redis `init` method.